edge-modules/edge-dashboard/src/modules.rs

```diff
@@ -0,0 +1,35 @@
+use url::Url;
+use edgelet_http_mgmt::*;
+use edgelet_core::{Module, ModuleRuntime};
+use futures::future::Future;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct SparseModule {
+    name: String,
+}
+
+pub fn get_management_uri(contents: &str) -> Option<String> {
+    let start_pattern = "management_uri: ";
+    let start = contents.find(start_pattern)? + start_pattern.len();
+    let end = contents.find("workload_uri: ")?;
+    let pre = contents[start..end].trim();
+    Some(pre[1..pre.len()-1].to_string())
+}
+
+pub fn get_list(uri: &str) -> Result<Vec<SparseModule>, url::ParseError> {
```

Comment:
Is this a test function that somehow made its way here? If yes, let's remove this.